src/Models/ProductModel.php

```diff
@@ -18,4 +19,31 @@ public function __construct($db)
     {
         $this->db = $db;
     }
+
+    public function addProduct(ProductEntityInterface $productEntity)
+    {
+        $array = ["sku"=>$productEntity->getSku(), "name"=>$productEntity->getName(),
+            "price"=>$productEntity->getPrice(), "stockLevel"=>$productEntity->getStockLevel()];
+
+        $query = $this->db->prepare("UPDATE `products`
+                                    SET `sku` = :sku
+                                        `name` = :name
+                                        `price` = :price
+                                        `stockLevel` = :stockLevel
+        ");
+        $result = $query->execute($array);
+        return $result;
+    }
+
+    public function checkProductExists(string $sku): bool
+    {
+        $query = $this->db->prepare("SELECT `id` FROM `products` WHERE `sku` = '$sku'");
+        $result = $query->execute();
```

Comment:
maybe do this instead?
```suggestion
        $query = $this->db->prepare("SELECT `id` FROM `products` WHERE `sku` = ?");
        $result = $query->execute([$sku]);
```